Data/src/test/java/org/tribuo/data/columnar/processors/response/MultioutputResponseProcessorTest.java

```diff
@@ -0,0 +1,142 @@
+package org.tribuo.data.columnar.processors.response;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.tribuo.Example;
+import org.tribuo.Output;
+import org.tribuo.data.columnar.FieldProcessor;
+import org.tribuo.data.columnar.ResponseProcessor;
+import org.tribuo.data.columnar.RowProcessor;
+import org.tribuo.data.columnar.processors.field.IdentityProcessor;
+import org.tribuo.data.csv.CSVDataSource;
+import org.tribuo.test.MockMultiOutput;
+import org.tribuo.test.MockMultiOutputFactory;
+import org.tribuo.test.MockOutputFactory;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MultioutputResponseProcessorTest {
+
+    private URI dataFile;
+    private URI missingDataFile;
+    private Map<String, FieldProcessor> fieldProcessors;
+    private List<Set<String>> multiTruthValues;
+
+
+    private static <T extends Output<T>> RowProcessor<T> makeRowProcessor(ResponseProcessor<T> responseProcessor, Map<String, FieldProcessor> fieldProcessors) {
+        return new RowProcessor<T>(responseProcessor, fieldProcessors);
+    }
+
+    private <T extends Output<T>, Label> void doTest(ResponseProcessor<T> responseProcessor, List<Label> expectedLabels, Function<T, Label> labelMapper) {
```

Comment:
Could you rename this type parameter from `Label` to something else, preferably all caps. I spent quite a while staring at it trying to figure out how it had access to the `Label` type given we're in Core.